### PR TITLE
Full project refactorings

### PR DESCRIPTION
**Pull request by Hellebore**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,64 +1,80 @@
...
-                "sourcery.metricsEnabled": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Show code metrics in the editor."
-                }
-            }
+    "menus": {
+      "explorer/context": [
+        {
+          "when": "explorerResourceIsFolder == true",
```
Comment (brendanator) [inline package.json:39]: Can it be done on a file too?

File: package.json [hunk 2]
```diff
@@ -1,64 +1,80 @@
...
+          "command": "sourcery.refactor.workspace",
+          "group": "navigation"
```
Comment (brendanator) [inline package.json:41]: I don't know if this is the setting that chooses it but appearing right at the very top of the menu above `New File` seems to be too prominent to me?

Comment (Hellebore) [reply package.json:41]: good catch - moved down to the next section

File: package.json (hunk 1 above)
Comment (Hellebore) [reply package.json:39]: done

**Pull request merged**